docs/quickstart.md

```diff
@@ -263,7 +267,7 @@ select count(*) from hive.default.hoodie_test
 
 
 
-## Incremental Queries
+## Incremental Queries of a Hoodie dataset
```

Comment:
is that really needed? is nt "Hoodie dataset" obvious from contexT?

Reply:
We generally mention a hoodie dataset everywhere so just added this for being standardized